client/src/components/Upload.js

```diff
@@ -244,18 +140,19 @@ const Upload = ({ setUpdateImages }) => {
 					<option value="illustrations">Illustrations</option>
 					<option value="logos">Logos</option>
 					<option value="icons">Icons</option>
-				</select>
+				</select>*/}
 			</div>
 
 			<button className="submitbtn">submit</button>
 
-			{/* Render the filtered list */}
+			{/* Render the filtered list 
 			<ul className="ulprinter">
 				{filteredList.map((item) => (
 					<li key={item.id}>{item.name}</li>
 				))}
-			</ul>
+				</ul> */}
```

Comment:
You can go through and just delete the code that's being commented out here. It keeps the codebase tidier